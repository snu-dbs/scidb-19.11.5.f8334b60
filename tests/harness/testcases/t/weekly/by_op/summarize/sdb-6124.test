--setup
create array public_RNAQUANTIFICATION <value:float> [dataset_version=0:*:0:1; dataset_id=0:*:0:10; measurementset_id=0:*:0:10; biosample_id=0:*:0:1000; feature_id=0:*:0:100]
--start-igdata
load(public_RNAQUANTIFICATION, '/public/data/tickets/sdb6124/public_opaque_save_RNAQUANT/public_RNAQUANTIFICATION', -2, 'opaque')
--stop-igdata

--test
summarize(filter(between(public_RNAQUANTIFICATION,null,null,null,null,null,null,null,null,null,null), measurementset_id = 16))

--cleanup
remove(public_RNAQUANTIFICATION)
