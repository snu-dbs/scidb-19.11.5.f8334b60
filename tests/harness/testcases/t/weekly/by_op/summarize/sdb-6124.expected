Query was executed successfully

[Query was executed successfully, ignoring data output by this query.]

{inst,attid} att,count,bytes,chunks,min_count,avg_count,max_count,min_bytes,avg_bytes,max_bytes
{0,0} 'all',3001110,84352184,14294,0,419.912,2790,24,5901.23,66984

Query was executed successfully

